resources/web3_init.js

```diff
@@ -107,13 +107,14 @@ StatusHttpProvider.prototype.isConnected = function () {
 }
 
 var protocol = window.location.protocol
+var address = providerAddress || "http://localhost:8545";
 console.log(protocol);
 if (typeof web3 === "undefined") {
     if (protocol == "https:") {
         console.log("StatusHttpProvider");
-        web3 = new Web3(new StatusHttpProvider("http://localhost:8545"));
+        web3 = new Web3(new StatusHttpProvider(address));
```

Comment:
This requires some changes in `react-native-webview-bridge` as atm `http://localhost:8545` is hardcoded in Java/Objc.